src/modules/theme.js

```diff
@@ -123,6 +123,7 @@ export default createMuiTheme({
   },
   shape: {
     borderRadius: 5,
+    searchBorderRadius: 35,
```

Comment:
This depends on the height of the searchBox...might make sense to set it to `100%` which has the same effect

Reply:
100% for the border-radius? That looks like this:
![image](https://user-images.githubusercontent.com/13254229/150814010-5be98447-cc4f-468d-862d-7a76c2982f7d.png)

Reply:
But, I can pull '35px' out of theme.

Reply:
Hahaha yikes!  Clearly that's not the way :) Ah wait, I'm getting mixed up with creating css circles...never mind